test/extended/builds/pipeline.go

```diff
@@ -200,7 +201,7 @@ var _ = g.Describe("[builds][Slow] openshift pipeline build", func() {
 			br.AssertSuccess()
 
 			g.By("confirm all the log annotations are there")
-			_, err = jenkins.ProcessLogURLAnnotations(oc, br)
+			err = jenkins.ProcessLogURLAnnotations(oc, br, !useSnapshotImage)
```

Comment:
if we don't have an easy way to tell which jenkins image we're testing, yeah i'm ok w/ just not doing that testing during PR testing for now and circling back to re-enable that check after we deprecate v1.